.github/workflows/import/poll-logs.js

```diff
@@ -237,20 +243,36 @@ async function main() {
             console.log(
               `Progress: Success: ${result.success} | Error: ${result.error}`
             );
-            
         })
-        .catch(() => {
+        .catch((e) => {
```

Comment:
The linter inside of me is asking: doesn't this lead to a "defined but never used" type of linter error?

Reply:
Removed